osquery/tables/networking/darwin/wifi.mm

```diff
@@ -99,10 +97,12 @@ void parseNetworks(const CFDictionaryRef& network, QueryData& results) {
     auto key = CFStringCreateWithCString(kCFAllocatorDefault, kv.second.c_str(),
                                          kCFStringEncodingUTF8);
     CFTypeRef value = nullptr;
-    if (key != nullptr && CFDictionaryGetValueIfPresent(network, key, &value)) {
-      r[kv.first] = extractNetworkProperties(value);
+    if (key != nullptr) {
```

Comment:
If `analyze` hits a snag and you fix it, you don't want to restart at `distclean`. It's pretty simple to add a nightly build on Jenkins to run the `make analyze` from scratch and open an issue if it fails. Then these little buggers will at most live for 24h.